.changeset/two-ties-fix.md

```diff
@@ -0,0 +1,6 @@
+---
+'@finos/legend-application-query-bootstrap': patch
+'@finos/legend-extension-dsl-data-space': patch
+'@finos/legend-graph': patch
+'@finos/legend-art': patch
```

Comment:
Maybe `legend-art` should be separated into its own changeset but I'm not positive